src/syntax-tree/ast.ts

```diff
@@ -1944,6 +1948,10 @@ export class IdentifierTkn extends Token implements TextEditable {
             return false;
         }
     }
+
+    setIdentifierText(text: string){
```

Comment:
This method is needed in order to set the text of a <code>VarAssignmentStmt</code> that is not actually part of the Module. It is not visible within the editor and so must not call <code>rebuid()</code>. Therefore, the existing <code>setEditedText()</code> could not have been used.

Reply:
I see. Thanks for explaining this.